Sources/Parse/Lexer.swift

```diff
@@ -130,12 +133,14 @@ enum TokenKind: Equatable {
     case .identifier(let value): return value
     case .unknown(let char): return char
     case .char(let value): return String(UnicodeScalar(value))
+    case .stringInterpolationLiteral(_): return "literal w/ interp"
```

Comment:
Cant implement this is-is, since the current data structure doesn't distinguish between interpolated & non-interpolated parts of the string literal